tensorflow/go/session.go

```diff
@@ -349,6 +350,79 @@ func (o *SessionOptions) c() (ret *C.TF_SessionOptions, done func(), err error)
 	}, nil
 }
 
+// JitLevel represents the level of optimization that the XLA compiler
+// performs during just-in-time compilation.
+type JitLevel int
+
+const (
+	// DEFAULT is the default setting for this version of TensorFlow,
+	// Currently the default is OFF, but it will change to ON in a future
+	// version.
+	DEFAULT JitLevel = 0
```

Comment:
Minor nit: Let's use mixed case naming for these as is more idiomatic in go (https://golang.org/doc/effective_go.html#mixed-caps and constants in standard library package like https://golang.org/pkg/net/http/#pkg-constants). Also "Default", "On", and "Off" are pretty generic names, how about:

```go
const (
  JitDefault JitLevel = 0
  JitOn JitLevel = 1
  JitOff JitLevel = -1
)
```

Reply:
Sure, no problem. Making those changes now.